modules/running-compliance-scans.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans_{context}"]
+= Running compliance scans
+
+Running a scan using the CIS profiles. For convenience, the Compliance Operator creates a `ScanSetting` object with reasonable defaults on startup. This `ScanSetting` object is named `default`.
+
+.Procedure
+
+.  Inspect the `ScanSetting` object by running:
++
+[source,terminal]
+----
+$ oc describe scansettings default
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSetting
+metadata:
+  name: default
+  namespace: openshift-compliance
+rawResultStorage:
+  pvAccessModes:
+  - ReadWriteOnce <1>
+  rotation: 3<2>
+  size: 1Gi <3>
+roles:
+- worker <4>
+- master <4>
+scanTolerations: <5>
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+  operator: Exists
+schedule: 0 1 * * *<6>
+----
+<1> The Compliance Operator creates a persistent volume (PV) that contains the results of the scans. By default, the PV will use access mode `ReadWriteOnce` because the Compliance Operator cannot make any assumptions about the storage classes configured on the cluster and `ReadWriteOnce` is a safe assumption that should be available on most clusters. If you need to fetch the scan results (link to Obtaining CO raw results…), you’ll do so using a helper pod, which also binds the volume. Volumes that use the `ReadWriteOnce` access mode can only be mounted by one pod at time, so it is important to remember to delete the helper pods. Otherwise, the Compliance Operator will not be able to reuse the volume for subsequent scans.
+<2> The Compliance Operator keeps results of three subsequent scans in the volume; older scans are rotated.
+<3> The Compliance Operator will allocate one GB of storage for the scan results.
+<4> If the scan setting uses any profiles that scan cluster nodes, scan these node roles.
+<5> The default scan setting binding also scans the master nodes
+<6> The default scan setting binding runs scans periodically every hour.
```

Comment:
This is ScanSetting object. In point 5 & 6, it is mentioned `scan setting binding`, please make correction.

Reply:
Also ScanSetting runs scans at 01:00 everyday not periodically every hour.